src/System.Net.Http.Listener/src/System/Net/HttpKnownHeaderNames.cs

```diff
@@ -0,0 +1,73 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net
+{
+    internal static class HttpKnownHeaderNames
```

Comment:
Maybe just use https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Net/HttpKnownHeaderNames.cs ?

The ordering looks different between the two files. Need to verify it's the same set.